python-client/giskard/models/base/__init__.py

```diff
@@ -25,6 +22,12 @@
 from giskard.path_utils import get_size
 from giskard.settings import settings
 
+try:
```

Comment:
as discussed, we install mlflow-skinny by default, so no need for this try-except

Reply:
we said though that we're going to do that, and raise an error with a helpful tip about `pip install <missing-package>`  (https://linear.app/giskard/issue/GSK-944/add-nicer-error-message-if-user-tries-to-import-module-with-missing), I'll take care of it in this branch.

Reply:
actually `mlflow-skinny` is our core dependency, without it `giskard` can't be installed, so we can be sure that it's available (which is indeed not the case for `torch`, `tensorflow` and `transformers` for example). So the check for `mlflow` isn't required